kernel/api/src/main/java/org/sakaiproject/tasks/api/Task.java

```diff
@@ -86,5 +86,7 @@ public class Task implements PersistableEntity<Long> {
     
     @Column(name = "TASK_OWNER", length = 99)
     private String owner;
-    
+
+    @Column(name = "SOFT_DELETED")
```

Comment:
thinking DELETED would be just as good!

Reply:
but I wonder do tasks really need to marked as deleted? vs actually deleting them?
maybe so I am just asking?